ipa-core/src/protocol/ipa_prf/malicious_security/verifier.rs

```diff
@@ -370,4 +401,30 @@ mod test {
 
         assert_eq!(q_final_another_way.as_u128(), EXPECTED_Q_FINAL);
     }
+
+    #[test]
+    fn differences_are_zero() {
+        const ZKP_1_and_2_LEFT: [u128; 14] = [0, 0, 13, 17, 11, 25, 7, 11, 25, 17, 9, 22, 23, 3];
+        const ZKP_1_and_2_RIGHT: [u128; 14] = [0, 30, 16, 13, 25, 3, 6, 1, 12, 29, 30, 7, 7, 3];
+        const OUT_LEFT: u128 = 27;
+        const OUT_RIGHT: u128 = 0;
+
+        let zkp_left = make_chunks::<Fp31, 7>(&ZKP_1_and_2_LEFT);
```

Comment:
Interesting. I associate the "make chunks" function with generating the UV iterations. But I suppose it works to generate a list of proofs too...

Reply:
It would be nice to somehow re-use the existing constants, maybe by leveling them up a bit in scope so that multiple methods can all use them. Then we could at least do something more readable like:
[PROOF_1, PROOF_2].map(...).collect()